internal/email/service.go

```diff
@@ -0,0 +1,206 @@
+package email
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"html/template"
+	"os"
+	"path/filepath"
+
+	"go.uber.org/zap"
+)
+
+// EmailService handles email operations
+type Email struct {
+	client *EmailClient
+	logger *zap.SugaredLogger
+}
+
+// NewEmailService creates a new email service
+func NewEmail(client *EmailClient, logger *zap.Logger) *Email {
+	return &Email{
+		client: client,
+		logger: logger.Sugar(),
+	}
+}
+
+// SendEmail sends a plain text email
+func (s *Email) SendEmail(ctx context.Context, req SendEmailRequest) (*SendEmailResponse, error) {
+	if !s.client.IsEnabled() {
+		s.logger.Warnw("email client is disabled, skipping email send",
+			"to", req.ToAddress,
+			"subject", req.Subject,
+		)
+		return &SendEmailResponse{
+			Success: false,
+			Error:   "email client is disabled",
+		}, nil
+	}
+
+	// Prioritize env var from address over request from address
+	fromAddress := s.client.GetFromAddress()
+	if fromAddress == "" {
+		fromAddress = req.FromAddress
+	}
```

Comment:
should be the other way round I guess? i.e prioritise the req data over system defaults from env